api/envoy/api/v2/endpoint/endpoint.proto

```diff
@@ -132,4 +132,12 @@ message LocalityLbEndpoints {
   //
   // Priorities should range from 0 (highest) to N (lowest) without skipping.
   uint32 priority = 5 [(validate.rules).uint32 = {lte: 128}];
+
+  // Optional: Per locality proximity value which indicates how close this
```

Comment:
Nit: I think most people will assume smaller values mean closer proximity, but perhaps it should be explicitly stated?

Reply:
does the comment "(lower the value, closer it is to the source locality)" convey this information ?

Reply:
Oh, I'm sorry. It does say exactly what I wanted.